src/models/Offer.js

```diff
@@ -84,6 +84,7 @@ const OfferSchema = new Schema({
     ownerName: { type: String, required: true },
 
     location: { type: String, required: true },
+    requirements: { type: [String], required: true },
```

Comment:
This only validates that the present field is an array, so empty arrays would be valid. Check this: https://mongoosejs.com/docs/migrating_to_5.html#array-required